doc/models/conductance.rst

```diff
@@ -0,0 +1,5 @@
+Conductance-based models
```

Comment:
```suggestion
Neuron models with conductance-based synapses
```
We cannot use "conductance-based models" because this is the dedicated expression for HH-like models (see [Scholarpedia article](http://www.scholarpedia.org/article/Conductance-based_models)